ios/FluentUI/AvatarGroup/MSFAvatarGroup.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// UIKit wrapper that exposes the SwiftUI AvatarGroup implementation.
+@objc public class MSFAvatarGroup: ControlHostingContainer {
+
+    /// Creates a new MSFAvatarGroup instance.
+    /// - Parameters:
+    ///   - style: The MSFAvatarGroupStyle value used by the AvatarGroup.
+    ///   - size: The MSFAvatarSize value used by the Avatars that will compose the AvatarGroup.
+    @objc public init(style: MSFAvatarGroupStyle,
+                      size: MSFAvatarSize) {
+        let avatarGroup = AvatarGroup(style: style,
+                                     size: size)
+        state = avatarGroup.state
+        super.init(AnyView(avatarGroup))
+    }
```

Comment:
Looks like I need to add back in something along the lines of view.backgroundColor = UIColor.clear. Didn't notice that until I uploaded the screenshots, though.

Reply:
I wonder if this is something that should be moved to the superclass; presumably most (all?) SwiftUI wrapper views would want that too?